README.md

```diff
@@ -1,36 +1,22 @@
 # GitHub CLI
 
-`gh` is GitHub on the command line, and it's now available in beta. It brings pull requests, issues, and other GitHub concepts to
-the terminal next to where you are already working with `git` and your code.
+`gh` is GitHub on the command line. It brings pull requests, issues, and other GitHub concepts to the terminal next to where you are already working with `git` and your code.
 
 ![screenshot of gh pr status](https://user-images.githubusercontent.com/98482/84171218-327e7a80-aa40-11ea-8cd1-5177fc2d0e72.png)
 
 ## Availability
 
-While in beta, GitHub CLI is available for repos hosted on GitHub.com only. It currently does not support repositories hosted on GitHub Enterprise Server or other hosting providers. We are planning on adding support for GitHub Enterprise Server after GitHub CLI is out of beta (likely towards the end of 2020), and we want to ensure that the API endpoints we use are more widely available for GHES versions that most GitHub customers are on.
+GitHub CLI is available for repos hosted on GitHub.com and GitHub Enterprise Server 2.20+. It is available to install on MacOS, Windows, and Linux. 
```

Comment:
1. There are now two sentences in a row that start with “GitHub CLI is available…”, so what I would suggest is massaging it a little bit to eliminate repetition.
2. (nitpick) “macOS”